src/options.zig

```diff
@@ -1685,6 +1693,10 @@ pub const BundleOptions = struct {
 
         opts.conditions = try ESMConditions.init(allocator, Target.DefaultConditions.get(opts.target));
 
+        if (transform.conditions.len > 0) {
+            opts.conditions.append(transform.conditions) catch unreachable;
```

Comment:
```suggestion
            opts.conditions.append(transform.conditions) catch bun.outOfMemory();
```
i know we already do this but `unreachable` on a memory allocation is dangerous.